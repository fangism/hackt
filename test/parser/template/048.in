// "048.in" should fail, no default relaxed parameters
// copy-modified from "044.in"

template <pint Q=5><pint R>
defproc foo();

foo<11><> bar;	// should die, insufficient default parameters

