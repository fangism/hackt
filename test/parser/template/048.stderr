At: "-stdin-":7:
parse error: syntax error
parser stacks:
state	value
#STATE#	(null) 
#STATE#	list<(root_item)>: (process-prototype) ... [5:1..13]
#STATE#	(qualified-id) [7:1..3]
#STATE#	list<(expr)>: (array-concatenation) ... [7:4..7]
#STATE#	< [7:8]
#STATE#	> [7:9]
in state #STATE#, possible rules are:
	expr_list_in_angles : '<' . expr_list '>'  (#RULE#)
acceptable tokens are: 
	'!' (shift)
	'(' (shift)
	'-' (shift)
	'~' (shift)
	ID (shift)
	FLOAT (shift)
	INT (shift)
	STRING (shift)
	SCOPE (shift)
	BOOL_TRUE (shift)
	BOOL_FALSE (shift)
